### DISCUSSION TITLE
A little indicator if the sound gets played earlier or later in the sound offset settings

### DISCUSSION THREAD

**Discussion by Schweeeeeeeeeeeeeeee** [ppy/osu#21339] [Ideas] [answered]
I'm never sure which direction I need to offset the sound when my hit error is 19 ms or so early, a little indicator would help :)
![image](https://user-images.githubusercontent.com/88291316/202923556-62b3131c-d20e-4c77-948f-95c7be6a032d.png)

**Comment by peppy** [comment, marked as answer]
Completely agree with this. Along with a lot of other settings UI, this would get a custom implementation in the future.

Out of curiosity do you find the display of the beatmap offset (the graph display that will show after retrying) ample in terms of understanding directionality? We spends quite a while discussing and reaching the conclusion there.

**Reply by Schweeeeeeeeeeeeeeee** [reply]
I think it's easy to understand because of the keywords like late and early.
The design looks great, thought I don't think it's fitting to the current style of Lazer.
:)
